### DISCUSSION TITLE
Sounds from Silence: audio in crouton without using cras

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#4969] [General]
Yes, you can definitely get sound in crouton without using cras, all you need is an external sound "card"; specifically, a simple USB audio adapter like this:

[SABRENT USB audio adapter](https://www.amazon.com/Sabrent-External-Adapter-Windows-AU-MMSA/dp/B00IRVQ0F8/)

The above device is USB 3.0 and is super fast, great if you're setting up a DAW. Even though I already owned a similar device, I ordered this one to test it out before recommending it online. And it works perfectly. Here is what you need to know:

First, install your crouton chroot using the "silence" branch:

`sudo CROUTON_BRANCH=silence crouton -r focal -t core,cli-extra,x11,gtk-extra,xorg,keyboard,extension,xfce`

Plug the adapter into a USB port, and run an audio cable from the green speaker port to your computer speaker, or alternatively, if you have wired headphones, plug its cord into the green speaker port. The port is a standard 3.5mm stereo port.

Next, launch your chroot: `sudo startxfce4`

Once you're at your Linux desktop, open a terminal and enter this command:

`cat /proc/asound/cards`

You will see a new device called Device (literally).

 Now create a file called .asoundrc which will specify the default alsa device.

`nano .asoundrc`

Here is what goes into the file:

---

pcm.!default {
    type hw
    card Device
}

ctl.!default {
    type hw
    card Device
}

---

Now you will have to manually install alsa-utils:

`sudo apt install alsa-utils`

At this point, you can tweak the settings of the sound card with `alsamixer`

What you do next is up to you. If you have a pencil mic, you can plug it into the pink mic port of the device. You can use `arecord`  and `aplay` to test out the device. You can listen to music with `lxmusic` or some other package. You can watch movies with `mplayer` or some other package.

What are the limitations? You will not be able to use a bluetooth headset. I'm not sure what other limitations there may be. Good luck, and update this discussion with your feedback.

Disclaimer: I do not have an Amazon seller account. I get no royalties from mentioning this product. I stripped off the cookies code from the Amazon link, so all you get is the product itself and nothing about me. AFAIK, the link is completely safe. However, click on it at your own discretion. Otherwise, you can search for "USB audio adapter".

My prior experience with Linux leads me to believe that you can use a number of different audio devices with crouton, e.g., an M-Audio preamp.

**Comment by Qwer-TeX** [comment]
THANK YOU SO MUCH. I will definitely remember the help I got from you :)

**Reply by Apocalypse10110** [reply]
I know that this thread has been inactive for a year but could you explain the "core,cli-extra,x11,gtk-extra,xorg,keyboard,extension,xfce" in the command you used to install crouton please. Thank you
